### PR TITLE
299 note EBS autoscaling deletion shortcomings on AWS batch CE page, FAQ

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Addressing https://support.seqera.io/a/tickets/2726 .

### CODE REVIEW

File: docs/compute-envs/aws-batch.md [hunk 1]
```diff
@@ -139,6 +139,9 @@ Once the AWS resources are set up, we can add a new **AWS Batch** environment in
 
 15. Select **EBS Auto scale** to allow the EC2 virtual machines to dynamically expand the amount of available disk space during task execution.
 
+    !!! warning "EBS autoscaling may cause unattached volumes on large clusters"
+        When running very large Batch clusters (around 10 000 vCPUs or more), AWS EC2 API rate limits may cause the deletion of unattached EBS volumes to fail. Volumes that remain active after Nextflow jobs have completed will incur additional costs, and should be manually deleted. Monitor your AWS account for any orphaned EBS volumes via the EC2 console, or with a Lambda function. See [here](https://aws.amazon.com/blogs/mt/controlling-your-aws-costs-by-deleting-unused-amazon-ebs-volumes/) for more information.
```
Comment (pditommaso) [inline docs/compute-envs/aws-batch.md:143]: Let's be more pessimist

```suggestion
        When running large Batch clusters (hundreds of compute nodes or more), AWS EC2 API rate limits may cause the deletion of unattached EBS volumes to fail. Volumes that remain active after Nextflow jobs have completed will incur additional costs, and should be manually deleted. Monitor your AWS account for any orphaned EBS volumes via the EC2 console, or with a Lambda function. See [here](https://aws.amazon.com/blogs/mt/controlling-your-aws-costs-by-deleting-unused-amazon-ebs-volumes/) for more information.  
```

File: docs/faqs.md [hunk 2]
```diff
@@ -790,6 +784,12 @@ As a best practice, Seqera suggests using Teams as the primary vehicle for assig
 
 ## Amazon
 
+### EBS 
+
+**<p data-question>Q: EBS Autoscaling: Why do some EBS volumes remain active after their associated jobs have completed?</p>**
+
+The EBS autoscaling solution relies upon an AWS-provided script running on each container host. This script performs AWS EC2 API requests to delete EBS volumes when the jobs using those volumes have been completed. When running very large Batch clusters (around 10 000 vCPUs or more), EC2 API rate limits may cause the deletion of unattached EBS volumes to fail. Volumes that remain active after Nextflow jobs have completed will incur additional costs, and should therefore be manually deleted. You can monitor your AWS account for any orphaned EBS volumes via the EC2 console, or with a Lambda function. See [here](https://aws.amazon.com/blogs/mt/controlling-your-aws-costs-by-deleting-unused-amazon-ebs-volumes/) for more information.
```
Comment (llewellyn-sl) [inline docs/faqs.md:791]:
```suggestion
The EBS autoscaling solution relies upon an AWS-provided script running on each container host. This script performs AWS EC2 API requests to delete EBS volumes when the jobs using those volumes have been completed. When running large Batch clusters (hundreds of compute nodes or more), EC2 API rate limits may cause the deletion of unattached EBS volumes to fail. Volumes that remain active after Nextflow jobs have been completed will incur additional costs and should therefore be manually deleted. You can monitor your AWS account for any orphaned EBS volumes via the EC2 console or with a Lambda function. See [here](https://aws.amazon.com/blogs/mt/controlling-your-aws-costs-by-deleting-unused-amazon-ebs-volumes/) for more information.
```

File: docs/compute-envs/aws-batch.md [hunk 3]
```diff
@@ -139,6 +139,9 @@ Once the AWS resources are set up, we can add a new **AWS Batch** environment in
 
 15. Select **EBS Auto scale** to allow the EC2 virtual machines to dynamically expand the amount of available disk space during task execution.
 
+    !!! warning "EBS autoscaling may cause unattached volumes on large clusters"
+        When running large Batch clusters (hundreds of compute nodes or more), AWS EC2 API rate limits may cause the deletion of unattached EBS volumes to fail. Volumes that remain active after Nextflow jobs have completed will incur additional costs, and should be manually deleted. Monitor your AWS account for any orphaned EBS volumes via the EC2 console, or with a Lambda function. See [here](https://aws.amazon.com/blogs/mt/controlling-your-aws-costs-by-deleting-unused-amazon-ebs-volumes/) for more information.
```
Comment (pditommaso) [inline docs/compute-envs/aws-batch.md:143]:
```suggestion
        When running large AWS Batch clusters (hundreds of compute nodes or more), AWS EC2 API rate limits may cause the deletion of unattached EBS volumes to fail. Volumes that remain active after Nextflow jobs have been completed will incur additional costs and should be manually deleted. Monitor your AWS account for any orphaned EBS volumes via the EC2 console, or with a Lambda function. See [here](https://aws.amazon.com/blogs/mt/controlling-your-aws-costs-by-deleting-unused-amazon-ebs-volumes/) for more information.  
```

Comment (llewellyn-sl) [inline docs/compute-envs/aws-batch.md:143]:
```suggestion
        When running large AWS Batch clusters (hundreds of compute nodes or more), EC2 API rate limits may cause the deletion of unattached EBS volumes to fail. Volumes that remain active after Nextflow jobs have completed will incur additional costs, and should be manually deleted. Monitor your AWS account for any orphaned EBS volumes via the EC2 console, or with a Lambda function. See [here](https://aws.amazon.com/blogs/mt/controlling-your-aws-costs-by-deleting-unused-amazon-ebs-volumes/) for more information.  
```

File: docs/faqs.md [hunk 4]
```diff
@@ -790,6 +784,12 @@ As a best practice, Seqera suggests using Teams as the primary vehicle for assig
 
 ## Amazon
 
+### EBS 
+
+**<p data-question>Q: EBS Autoscaling: Why do some EBS volumes remain active after their associated jobs have completed?</p>**
+
+The EBS autoscaling solution relies upon an AWS-provided script running on each container host. This script performs AWS EC2 API requests to delete EBS volumes when the jobs using those volumes have been completed. When running large Batch clusters (hundreds of compute nodes or more), EC2 API rate limits may cause the deletion of unattached EBS volumes to fail. Volumes that remain active after Nextflow jobs have been completed will incur additional costs and should therefore be manually deleted. You can monitor your AWS account for any orphaned EBS volumes via the EC2 console or with a Lambda function. See [here](https://aws.amazon.com/blogs/mt/controlling-your-aws-costs-by-deleting-unused-amazon-ebs-volumes/) for more information.
```
Comment (llewellyn-sl) [inline docs/faqs.md:791]:
```suggestion
The EBS autoscaling solution relies on an AWS-provided script running on each container host. This script performs AWS EC2 API requests to delete EBS volumes when the jobs using those volumes have been completed. When running large Batch clusters (hundreds of compute nodes or more), EC2 API rate limits may cause the deletion of unattached EBS volumes to fail. Volumes that remain active after Nextflow jobs have been completed will incur additional costs and should therefore be manually deleted. You can monitor your AWS account for any orphaned EBS volumes via the EC2 console or with a Lambda function. See [here](https://aws.amazon.com/blogs/mt/controlling-your-aws-costs-by-deleting-unused-amazon-ebs-volumes/) for more information.
```

**Pull request merged**